bundles/tampere/content-editor/view/SideContentEditor.js

```diff
@@ -662,19 +668,19 @@ Oskari.clazz.define('Oskari.tampere.bundle.content-editor.view.SideContentEditor
                 colourScheme,
                 font;
 
-            $('.content-draw-tools').removeClass('hide');
+            jQuery('.content-draw-tools').removeClass('hide');
```

Comment:
Limit the element search to the container of this functionality. Something like me.mainPanel.find('.content-draw-tools'). Same comment for similar lines below.